src/fred/cli/releases.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+"""
+FRED CLI - Releases Namespace.
+"""
+import json
+
+import click
+
+from .. import BaseFredAPIError
+from .._util import generate_api_kwargs
+
+
+__all__ = [
+    "releases",
+]
+
+
+@click.group()
+@click.pass_context
+def releases(ctx):
+    """
+    Releases CLI Namespace.
+    """
+    pass
+
+
+@releases.command()
+@click.argument("args", nargs=-1)
+@click.pass_context
+def get_releases(ctx, args: tuple):
+    """Get releases."""
+    try:
+        click.echo(json.dumps(ctx.obj["client"].get_releases(**generate_api_kwargs(args)), indent=4))
+    except (ValueError, BaseFredAPIError) as e:
+        click.echo(click.style(e, fg="red"))
+
+
+@releases.command()
+@click.argument("args", nargs=-1)
+@click.pass_context
+def get_releases_dates(ctx, args: tuple):
+    """Get a category."""
+    try:
+        click.echo(json.dumps(ctx.obj["client"].get_releases_dates(**generate_api_kwargs(args)), indent=4))
+    except (ValueError, BaseFredAPIError) as e:
+        click.echo(click.style(e, fg="red"))
+
+
+@releases.command()
+@click.option("--release-id", "-r", required=True, type=click.INT, help="Release ID.")
```

Comment:
Change this to use shortname `-i` for `--release-id`, and other usages for this arg as well

Reply:
Also this should be of click type `STRING`

Reply:
Please update other aspects that use this same click type